src/app/components/connect-applications-wizard/connect-applications-wizard.less

```diff
@@ -0,0 +1,69 @@
+#connect-apps-panel {
+  min-height: 85%;
+  max-height: 85%;
+  min-width: 90%;
+  overflow: auto;
+    
+  .title {
+    font-weight: @onboarding-title-weight;
+    color: @onboarding-title-color; 
+  }
+
+  .apps-container {
+    height: 100%;
+    padding: 0;
+    margin: 0;
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -webkit-flex;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .app-container {
+    flex-grow: 1;
+    width: 200px;
+    margin: 10px;
+
+    h4 {
+      color: @onboarding-connect-apps-app-name-color;
+      font-weight: @onboarding-connect-apps-app-name-weight;
+    }
+  }
+
+  img.media-object {
+    max-width: 140px;
+    height: auto;
+    margin: 0 auto;
+  }
+
+  .connect-apps-wrap {
+    margin-top: 20px;
+  }
+
+  .table-bordered {
+    border: none; 
+  }
+
+  .bottom-align-text {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+  }
+  
+  .data-sync-info {
+    overflow: auto;
+  }
+
+  .btn-primary {
```

Comment:
Instead of overriding this button class, can't you use another class?

This project uses semantic actions (cancel action: default button, info action: primary button, save/change action: warning button, delete action: danger button)